public/stylesheets/board.css

```diff
@@ -379,6 +379,29 @@ div.lichess_board {
 #top div.color_demo.maple {
   background-image: url(../images/board/maple.jpg);
 }
+
+/* Racing Kings board decorations */
+
+.racingKings .cg-board.orientation-white::after {
```

Comment:
Like the KOTH decorations, I think `::before` should be used instead, so the pseudo-element doesn't interfere with having the hover event reach the underlying squares. See #976